src/promptflow-devkit/promptflow/_sdk/_pf_client.py

```diff
@@ -72,6 +72,7 @@ def run(
         variant: str = None,
         connections: dict = None,
         environment_variables: dict = None,
+        properties: dict = None,
```

Comment:
@singankit Please use this _run() function which exposes the properties parameter. We hide it from the public API.

Reply:
Updated the usage in pr description